parquet-column/src/main/java/org/apache/parquet/io/RecordReaderImplementation.java

```diff
@@ -18,15 +18,8 @@
  */
 package org.apache.parquet.io;
 
-import java.util.ArrayList;
```

Comment:
Minor: reordering imports make it difficult to move patches around and backport things.